topologies/kne/ixia-arista-ixia.textproto

```diff
@@ -0,0 +1,70 @@
+name: "ixia-c"
```

Comment:
Drop the 2 port variant and keep the 4 port one?

Reply:
We could...although the 4 ports does require more resources...